ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/RedirectSSOTest.kt

```diff
@@ -119,7 +117,41 @@ class RedirectSSOTest : StringSpec() {
                     getServiceProvider(IdpSSOResponder::class).getResponseForRedirectRequest(
                             response)
             val samlResponseDom = finalHttpResponse.getBindingVerifier().decodeAndVerify()
-            val samlRequestDom = authnRequest.dom as Node
+
+            // Main goal of this test is to verify the ACS in verifyAssertionConsumerService
+            CoreAuthnRequestProtocolVerifier(authnRequest, samlResponseDom).apply {
+                verify()
+                verifyAssertionConsumerService(finalHttpResponse)
+            }
+            SingleSignOnProfileVerifier(authnRequest, samlResponseDom).apply {
+                verify()
+                verifyBinding(finalHttpResponse)
+            }
+        }
+
+        "Redirect AuthnRequest Using DSA1 Signature Algorithm" {
+            Log.debugWithSupplier { "Redirect AuthnRequest Using DSA1 Signature Algorithm" }
+            val authnRequest = createDefaultAuthnRequest(SamlProtocol.Binding.HTTP_REDIRECT).apply {
+                assertionConsumerServiceURL = null
+            }
+
+            val encodedRequest = encodeAuthnRequest(authnRequest)
+            // Main goal of this test is to use DSA signature algorithm to verify that the IdP
```

Comment:
✏️ It may be useful to add these "main goal" comments as a javadoc of the test. We are basically repeating the test name.